stacks/config.py

```diff
@@ -0,0 +1,44 @@
+import os
+import os.path
+import botocore.config
+
+from .utils import load_yaml
+
+AWS_CONFIG_FILE = os.environ['HOME'] + '/.aws/credentials'
+DEFAULT_REGION = 'us-east-1'
```

Comment:
Maybe we shouldn't set a default region any more? the AWS ruby sdk no longer assumes anything.